scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -152,3 +153,29 @@ def __iter__(self):
 
     def get_generated_keys(self) -> List[str]:
         return [self._name]
+
+
+class AttackPointIteratorRepeat(AttackPointIterator):
+    """
+    Attack point iterator class that iterates 
+    over a configuration a repeated amount of times.
+    """
+
+    def __init__(self, operation: str, repetitions: int,
+                 configuration: Dict) -> None:
+        """Initialize the repeated iterate."""
+        assert "repeat" == operation
+        self._repetitions = repetitions
+        self._configuration_iterator = build_attack_points_iterator(
+            configuration)
+        self._len = repetitions * len(self._configuration_iterator)
+
+    def __len__(self) -> int:
+        return self._len
+
+    def __iter__(self):
+        return iter(value for repetition in range(self._repetitions)
```

Comment:
NIT Two forloops nested in iter expression are not the most readable (not clear which happens first). But ok with me if you are 100% sure it is correct.